src/xc_integrator/local_work_driver/host/obara_saika/src/config_obara_saika.hpp

```diff
@@ -252,7 +252,7 @@ namespace XCPU {
 
 // Scalar SIMD Emulation
 #else
-#ifdef __GNUC__
+#if (defined(__GNUC__) && __cplusplus >= 202302L)
```

Comment:
Not sure I understand the `__cplusplus` guard here, is there a compiler this was erroring out for without it. Honestly, we can remove this warning, it was more for internal use anyways. However, you *can* make it go away by ensuring that the host compilation uses arch specific flags (e.g. `-march=native`). I don't include them by default because there's not a good (portable) way to do it, but if you're seeing this warning for a `Release` build, you should rethink how you're compiling.

Reply:
I'm compiling with Apple Clang 17 and it complains that `#warning` is a C++23 feature. The `__cplusplus` guard checks that the C++ standard is C++23 or greater. That said, I'll just remove it.